.github/workflows/unit-int.yml

```diff
@@ -111,7 +117,7 @@ jobs:
         if: always()
         run: nix-shell --run "./scripts/clean-cargo-tests.sh"
 # debugging
-      # - name: Setup tmate session
-      #   if: ${{ failure() }}
-      #   timeout-minutes: 240
-      #   uses: mxschmitt/action-tmate@v3
+      - name: Setup tmate session
+        if: ${{ failure() }}
+        timeout-minutes: 240
+        uses: mxschmitt/action-tmate@v3
```

Comment:
Comment these?

Reply:
No, I'm trying to debug the failures, it's not passing